### PR TITLE
Director is Changed

### PR DESCRIPTION
**Pull request by prakharvk**
Since the Director of IIT(ISM),Dhanbad is changed but i noticed that the placement page was still showing the name of old Director along with the pic of new Director. So, updated the name of the Director.

### CODE REVIEW

File: placement_home.html [hunk 1]
```diff
@@ -123,7 +123,7 @@ <h3 style="font-family: 'Merriweather', serif;", cursive;>INDIAN INSTITUTE OF TE
     <section class="row text-center placeholders">
         <div class="col-6 col-sm-3 placeholder">
             <a href="#mess_dir"><img src="http://iitism.ac.in/assets/img/admin/dt.jpg" width="200" height="200" class="img-fluid rounded-circle" alt="Generic placeholder thumbnail"></a>
```
Comment (nveenjain) [inline placement_home.html]: Thanks for the PR, could you also update the pic of director? Possibly bring it offline?

**Review by nveenjain — CHANGES_REQUESTED** [review]

**Comment by prakharvk** [comment]
I have done the specified changes, making all the images offline.

File: placement_home.html (hunk 1 above)
Comment (codeShaurya) [reply placement_home.html]: Welcome, @prakharvk! Congrats for being the first contributor to the project.

**Review by nveenjain — APPROVED** [review]
LGTM. Thanks

**Pull request merged**